comms/discovery/misc/chat_id.go

```diff
@@ -0,0 +1,13 @@
+package misc
+
+import "fmt"
+
+func ChatID(id1, id2 int) string {
```

Comment:
nit: maybe a docstring explaining the convention for 1:1 chats w/ a pointer to the SDK code doing the same?